s3tests_boto3/functional/test_iam.py

```diff
@@ -0,0 +1,211 @@
+import json
+from operator import eq
+
+from botocore.exceptions import ClientError
+from nose.plugins.attrib import attr
+
+from s3tests.functional.utils import assert_raises
+from . import get_tenant_iam_client, get_tenant_user_id
+from .utils import _get_status
+
+
+@attr(resource='user-policy')
+@attr(method='put')
+@attr(operation='Verify Put User Policy')
+@attr(assertion='succeeds')
+@attr('user-policy')
+def test_put_user_policy():
+    client = get_tenant_iam_client()
+
+    policy_document = json.dumps(
+        {"Version": "2012-10-17",
```

Comment:
Formatting of the dict assigned to the policy_document variable can be made consistent across the file.

Reply:
done